pkg/restic/path_win.go

```diff
@@ -0,0 +1,37 @@
+//go:build windows
+// +build windows
+
+package restic
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+// toPathFilter converts a directory path to a path filter. Restic requires
+// these to be absolute, starting with a forward slash. On Windows, we convert
+// the drive letter to a single character and prepend a forward slash.
+func toPathFilter(path string) string {
+	cleanedPath := filepath.Clean(path)
+	// If clean results in an empty string, ignoring the volume, it returns "."
+	if cleanedPath != path+"." {
```

Comment:
Hey, not sure I follow what this check is doing -- in general we'll never call toPathFilter with an input that that contains `/../` e.g. `/foo/../`. If we do run into that type of case, I think it'd be fine for that behavior to be unbefined.

Reply:
It's a weird one... this was for the `C:` test case. [`filepath.Clean`](https://pkg.go.dev/path/filepath#Clean) states that if the path is an empty string, it returns dot. Well... it doesn't include a drive volume in its test. So `C:` will return `C:.` It's odd, tbh: https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/path/filepath/path.go;l=101-103

We can probably ignore this edge case when moving this into the test.